api/src/BcGov.Malt.Web/BcGov.Malt.Web.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UserSecretsId>e50dd137-4459-4a4a-aed3-0e56f3723cff</UserSecretsId>
+    <AssemblyName>BcGov.Malt.Web</AssemblyName>
+    <RootNamespace>BcGov.Malt.Web</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="3.1.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.10" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.0" />
+    <PackageReference Include="Polly" Version="7.2.0" />
+    <PackageReference Include="Polly.Contrib.Simmy" Version="0.3.0" />
+    <PackageReference Include="Serilog" Version="2.9.0" />
+    <PackageReference Include="SerilogAnalyzer" Version="0.15.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="5.0.0" />
+  </ItemGroup>
+
+

```

Comment:
supernit: extra newline

Reply:
really?  The IDE modifies this file. I do not edit directly.

Reply:
these are just nits meaning that they are not mandatory to address and are not blockers in any way - I am just mentioning them for awareness and in case you would like to make the change 😄 not required though

Reply:
I know however, seems strange to point it out.  For example, in the javascript side of things, the [package.json](https://github.com/bcgov/maltd/blob/11fa8a4d7a67fb168e5b1795a59625e311cb9e41/maltd-frontend/package.json#L34) [package-lock.json](https://github.com/bcgov/maltd/blob/11fa8a4d7a67fb168e5b1795a59625e311cb9e41/maltd-frontend/package-lock.json#L14095) file is missing a new line at the end.  This is not typically a file that is edited manually.  But are we required to add the new line at the end every time these file are changed by npm?

Reply:
Sorry about that Phil, didn’t mean any harm. My bad